test/unit/eigensolver/test_eigensolver.cpp

```diff
@@ -0,0 +1,98 @@
+//
+// Distributed Linear Algebra with Future (DLAF)
+//
+// Copyright (c) 2018-2022, ETH Zurich
+// All rights reserved.
+//
+// Please, refer to the LICENSE file in the root directory.
+// SPDX-License-Identifier: BSD-3-Clause
+//
+#include "dlaf/eigensolver/eigensolver.h"
+
+#include <functional>
+#include <tuple>
+#include "gtest/gtest.h"
+#include "dlaf/matrix/matrix.h"
+#include "dlaf_test/matrix/matrix_local.h"
+#include "dlaf_test/matrix/util_matrix.h"
+#include "dlaf_test/matrix/util_matrix_local.h"
+#include "dlaf_test/util_types.h"
+
+using namespace dlaf;
+using namespace dlaf::comm;
+using namespace dlaf::matrix;
+using namespace dlaf::matrix::test;
+using namespace dlaf::test;
+using namespace testing;
+
+template <typename Type>
+class EigensolverTestMC : public ::testing::Test {};
+TYPED_TEST_SUITE(EigensolverTestMC, MatrixElementTypes);
+
+const std::vector<blas::Uplo> blas_uplos({blas::Uplo::Lower});
+
+const std::vector<std::tuple<SizeType, SizeType>> sizes = {
+    //    {0, 2},                              // m = 0
```

Comment:
what about the test cases? are we ok? should we add back also more empty cases?

Reply:
The empty case should be enabled. Not sure why it is currently not.
What do you mean with `more` empty cases?